### PR TITLE
String literals should not be duplicated

### PR DESCRIPTION
**Pull request by kotisyamala**
| Fixed Issues?            | String literals should not be duplicated
| Patch: Bug Fix?          |
| Minor: New Feature?      |
| Major: Breaking Change?  |
| Tests Added + Pass?      | Yes
| Documentation Provided   | Yes 
| Any Dependency Changes?  |
| License                  | Apache License, Version 2.0

Duplicated string literals make the process of refactoring error-prone, since you must be sure to update all occurrences.
On the other hand, constants can be referenced from many places, but only need to be updated in a single place.

### CODE REVIEW

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java [hunk 1]
```diff
@@ -190,7 +190,7 @@ private void sendResponse(InputStream stream, long size, String mimeType, String
             response.setContentLength((int) size);
         }
         if (inline) {
-            response.setHeader(CONTENT_DISPOSITION_HEADER, "inline");
+            response.setHeader(CONTENT_DISPOSITION_HEADER, INLINE_SELECTOR);
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java]: I would not use `INLINE_SELECTOR` as it represents a request selector, which has a different meaning than the content disposition header which is used here. You might want to introduce a new constant called e.g. `CONTENT_DISPOSTION_INLINE`.

**Comment by gabrielwalt** [comment]
@kotisyamala, Will you make the requested changes? Else, we'll close this PR.

**Comment by kotisyamala** [comment]
> @kotisyamala, Will you make the requested changes? Else, we'll close this PR.

Will raise the changes to this PR today.

Thanks,
Koti

**Comment by gabrielwalt** [comment]
Closing this for now, I'm not sure this is even a desired contribution.

**Pull request closed without merging**